oauth2_provider/models.py

```diff
@@ -216,6 +238,8 @@ def allow_scopes(self, scopes):
         """
         if not scopes:
             return True
+        if isinstance(scopes, six.string_types):
```

Comment:
well a string is in fact an iterable, using it by accident might show up weird errors, this is a bit more developer friendly in that it will do the thing that you wanted to happen when you pass a string. But you are also right that it might encourage the use of a single string and confuse more people when they see inconsitent usage of 'scopes'